exercises/concept/fundamentals/.docs/instructions.md

```diff
@@ -0,0 +1,66 @@
+# Instructions
+
+Once upon a time, there was a bustling city called Cyberspace.
+Cyberspace was a digital metropolis, with millions of inhabitants logging in and out every day.
+To keep track of all these users, Cyberspace had a special file called /etc/passwd.
+
+The /etc/passwd file was like a phonebook for the city.
+It listed every user who had an account on the system, along with some important information about each user:
+
+| n   | field |
```

Comment:
I'm not sure this will get rendered properly. You may need a codeblock for this. See saddle points.

Reply:
I'm basing it on https://exercism.org/docs/building/tracks/stories rendered from https://github.com/exercism/docs/blob/main/building/tracks/stories/README.md?plain=1